### PR TITLE
add Hong Kong Baptist University

### PR DESCRIPTION
**Pull request by xu-cheng**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -1567,6 +1568,7 @@ Byoungyoung Lee,Purdue University,https://www.cs.purdue.edu/people/faculty/lee25
 Byrav Ramamurthy,University of Nebraska,http://cse.unl.edu/~byrav/,Moj2TLMAAAAJ
 Byron Boots,Georgia Institute of Technology,http://www.cc.gatech.edu/~bboots3/,kXB8FBoAAAAJ
 Byron C. Wallace,Northeastern University,http://www.byronwallace.com/,KTzRHmwAAAAJ
+Byron Choi,Hong Kong Baptist University,http://www.comp.hkbu.edu.hk/~choi,S6L1eOMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: No such URL.

File: csrankings.csv [hunk 2]
```diff
@@ -7398,6 +7405,7 @@ Michael Kazhdan,Johns Hopkins University,http://www.cs.jhu.edu/~misha/,soUq_eQAA
 Michael Kearns,University of Pennsylvania,http://www.cis.upenn.edu/~mkearns/,8iQk0DIAAAAJ
 Michael Kifer,Stony Brook University,http://www3.cs.stonybrook.edu/~kifer/,HIOYWhYAAAAJ
 Michael Kirley,University of Melbourne,http://findanexpert.unimelb.edu.au/display/person26129/,te9wVHwAAAAJ
+Michael Kwok-Po Ng,Hong Kong Baptist University,http://http://www.math.hkbu.edu.hk/~mng,BBpjLiIAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: URL is wrong (http://http://).

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (xu-cheng) [reply csrankings.csv]: Fixed

File: csrankings.csv (hunk 2 above)
Comment (xu-cheng) [reply csrankings.csv]: Fixed

**Review by emeryberger — APPROVED** [review]

**Comment by xu-cheng** [comment]
Thanks for the merging. 🎉

**Pull request merged**